src/indra_cogex/client/neo4j_client.py

```diff
@@ -1469,3 +1533,28 @@ def do_cypher_tx(
     # run-time
     result = tx.run(query, parameters=query_params)
     return [record.values() for record in result]
+
+
+def do_cypher_tx_with_keys(
+        tx: ManagedTransaction,
+        query: str,
+        **query_params
+) -> tuple[List[str], List[List]]:
```

Comment:
Refactor these two functions so that we have one that runs tx.run and returns both keys and values, and another that just runs that function but drops the keys and only returns the values.